open-api/rest-catalog-open-api.yaml

```diff
@@ -2510,6 +2510,32 @@ components:
         default-sort-order-id:
           type: integer
 
+    ViewRequirement:
```

Comment:
Why is this separate? I thought that we were just going to add assert-view-uuid to the other. I don't like having separate View and Table requirements since we use the same ones in most places.

Reply:
I followed up with @nastra offline. Since the only requirement for views is assert-view-uuid and we don't want to change or reuse assert-table-uuid, it does make sense to have a separate set of requirements.